networks/rpc/server.go

```diff
@@ -51,8 +51,13 @@ const (
 	concurrencyLimit = 3000
 )
 
-// pendingRequestCount is a total number of concurrent RPC method calls
-var pendingRequestCount int64 = 0
+var (
+	// pendingRequestCount is a total number of concurrent RPC method calls
+	pendingRequestCount int64 = 0
+
```

Comment:
```suggestion
var (
```

Reply:
@KimKyungup Ths suggestion removes `pendingRequestCount `. Do you want to declare `pendingRequestCount` `MaxSubscriptionPerConn` seperatly?

Reply:
It's my mistake. Please forget it. :)